src/App.test.tsx

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
```

Comment:
When running `npm run test a` this test is failing. We should fix the test or remove it.

Reply:
I think i changed the test so it should pass now